tests/test_forms/test_forms.py

```diff
@@ -0,0 +1,68 @@
+from django.test import TestCase
+
+from taxi.forms import CarForm, DriverLicenseUpdateForm
+from taxi.models import Driver, Manufacturer
+
+
+class TestForm(TestCase):
+    def setUp(self):
+        self.manufacturer = Manufacturer.objects.create(
+            name="Fasd",
+            country="asd"
+        )
+        self.driver1 = Driver.objects.create_user(
+            username="Driver1",
+            password="1234",
+            license_number="TES12345"
+        )
+        self.driver2 = Driver.objects.create_user(
+            username="Driver2",
+            password="1234",
+            license_number="TES12341"
+        )
+
+    def test_valid_data_car(self):
+        form_data = {
+            "model": "Test10",
+            "manufacturer": 1,
```

Comment:
The `manufacturer` field in `form_data` is set to `1`, which assumes the ID of the manufacturer. Instead, use `self.manufacturer.id` to ensure the correct ID is used.